addons/binding/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/internal/hw/HwSerialBridgeHandler.java

```diff
@@ -0,0 +1,275 @@
+/**
```

Comment:
It seems that this file has a missing, outdated or otherwise non standard copyright header, can you fix all of them by running 

`mvn license:format` 

Please commit only the ones for your binding.